lib/constants/tableProps.js

```diff
@@ -4,59 +4,59 @@ const exampleTableData = [
     Age: '2m',
     '# of Tx': '2464',
     'BTC Sent': '18',
-    'Size (kB)': '1,018',
-    Weight: '1898',
-    'Avg Fee': '$1.58',
+    'Size (kB)': '1,311',
```

Comment:
what were these changed for?

Reply:
So that you could tell the difference between them when paginating, because every other is the same